config/service-brokers/aiven/config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "port": "8080",
+  "log_level": "info",
+  "cloud": "aws-eu-west-1",
```

Comment:
I don't think we should include this here any more now that this config value is taken from the environment, it will only cause confusion.